uamqp/message.py

```diff
@@ -389,13 +389,16 @@ def reject(self, condition=None, description=None):
         :type condition: bytes or str
         :param description: A description/reason to accompany the rejection.
         :type description: bytes or str
+        :param info: Information about the error condition.
+        :type info: dict
```

Comment:
So we call it description/reason in SB; is there a reason for the nomenclature split, or should we normalize?

Reply:
yeah, it's the spec of `reject` in amqp, SB is utilizing the `amqp.reject` for `sb.dead-letter`.

```
-- Copied from AMQP

<type name="rejected" class="composite" source="list" provides="delivery-state, outcome">
    <descriptor name="amqp:rejected:list" code="0x00000000:0x00000025"/>
    <field name="error" type="error"/>
</type>


<type name="error" class="composite" source="list">
    <descriptor name="amqp:error:list" code="0x00000000:0x0000001d"/>
    <field name="condition" type="symbol" requires="error-condition" mandatory="true"/>
    <field name="description" type="string"/>
    <field name="info" type="fields"/>
</type>

Field Details
condition - error condition
    A symbolic value indicating the error condition.
description - descriptive text about the error condition
    This text supplies any supplementary details not indicated by the condition eld.
    This text can be logged as an aid to resolving issues.
info - map carrying information about the error condition
```